src/main/resources/application.properties

```diff
@@ -1,5 +1,10 @@
 spring.application.name=spring-boot-application-template
-server.port=8080
+spring.profiles.active='@active.profile@'
```

Comment:
For the local development we have Testcontainers based configuration, which automatically spin up a Postgres docker container and DataSource properties will be automatically configured. So we don't need to explicitly configure the datasource properties. 

Take a look at `src/test/java/com/sivalabs/myapp/TestcontainersConfig.java` and `src/test/java/com/sivalabs/myapp/TestApplication.java`. If you have Docker installed, you can simply run `TestApplication.java` to start the application.